src/_gettsim/groupings.py

```diff
@@ -33,6 +39,61 @@ def bg_id_numpy(
     return numpy.asarray(result)
 
 
+def eg_id_numpy(
```

Comment:
Do we have checks for whether eg / ehe have size 1 or 2?

Reply:
Added issue #707